source/bits/cshiftr.ts

```diff
@@ -0,0 +1,19 @@
+import { Bits } from '../types'
+
+/**
+ * Circular Shift Right
+ *
+ * @example
+ * cshiftr([1,0,1,1,0,1]) => [1,1,0,1,1,0]
```

Comment:
Is this the common way to call this function in other languages or did you come up with this name? If it’s super common (and I’m just not aware of it) then I’ll consider this one. Otherwise, I’m more in favor of something like `.circularShiftRight` as it’s easier to understand.

Reply:
agree